ferrocene/tools/coverage-of-subset/src/main.rs

```diff
@@ -3,16 +3,32 @@
 
 use std::fs;
 use std::path::Path;
-use std::sync::LazyLock;
 
 /// Uncertified code is marked with this attribute
 const MARKER: &str = "#[cfg(not(feature = \"ferrocene_certified\"))]";
+const APPLICABLE: [&str; 18] = [
+    "fn",
+    "unsafe fn",
+    "const fn",
+    "const unsafe fn",
+    "pub fn",
+    "pub unsafe fn",
+    "pub const fn",
+    "pub const unsafe fn",
+    "pub(crate) fn",
+    "pub(crate) unsafe fn",
+    "pub(crate) const fn",
+    "pub(crate) const unsafe fn",
+    "impl",
+    "pub impl",
+    "pub(crate) impl",
+    "mod",
+    "pub mod",
+    "pub(crate) mod",
+];
```

Comment:
Let's add `unsafe impl`. At the moment this is not relevant, but it could become relevant e.g. for `unsafe impl<T: Clone> CopySpec for T` or `unsafe impl GenericRadix for $T`.
```suggestion
    "unsafe impl",
];
```